### DISCUSSION TITLE
v5.2 upcoming

### DISCUSSION THREAD

**Discussion by wnunezc** [twbs/bootstrap#35951] [General] [answered]
v5.2 upcoming:

Any idea if version 5.2 will come out before the end of March?

**Comment by mdo** [comment, marked as answer]
That'd be rad! Currently we need more eyes on open PRs to help review things. Have a few outstanding items we really need to lock in first.

**Reply by m-y-n-o-n-a** [reply]
i would also like to know as there is this bug with maps that is a big blocker

https://github.com/twbs/bootstrap/issues/34756
